thumbor/engines/__init__.py

```diff
@@ -147,6 +159,20 @@ def convert_svg_to_png(self, buffer):
         self.extension = EXTENSION.get(mime, '.jpg')
         return buffer
 
+    def convert_tif_to_png(self, buffer):
+        if not numpy:
+            msg = """[BaseEngin] convert_tif_to_png failed numpy not imported"""
+            logger.error(msg)
+            return buffer
+        if not cv2:
+            msg = """[BaseEngin] convert_tif_to_png failed opencv not imported"""
+            logger.error(msg)
+            return buffer
+
+        img = cv2.imdecode(numpy.fromstring(buffer), -1)
```

Comment:
`numpy.fromstring(buffer)` is complaining:
> string size must be a multiple of element size

So I tried setting second argument:
```python
numpy.fromstring(buffer, dtype='uint16')
```
and that works